util/string_view.h

```diff
@@ -175,6 +175,11 @@ std::string_view slice_suffix(std::string_view view, std::size_t size);
  */
 bool ends_with(std::string_view view, std::string_view suffix);
 
+/**
+ * Returns whether the view starts with the specified prefix.
+ */
+bool starts_with(std::string_view view, std::string_view prefix);
+
```

Comment:
nit. C++20 string_view supports `starts_with`. At this point we could consider requiring a C++20 compiler..

Reply:
yes, i agree - would be nice to move ahead to c++ 20.